src/rocker/extensions.py

```diff
@@ -186,6 +186,29 @@ def register_arguments(parser, defaults):
         parser.add_argument('--name', default=defaults.get('name', ''),
                             help='Name of the container.')
 
+class Detach(RockerExtension):
+    @staticmethod
+    def get_name():
+        return 'detach'
+    
+    def __init__(self):
+        self.name = Detach.get_name()
+    
+    def get_docker_args(self, cliargs):
+        args = ''
+        detach = cliargs.get('detach', False)
+        if detach:
+            args += ' --detach'
+        return args
+    
+    @staticmethod
+    def register_arguments(parser, defaults):
+        parser.add_argument(
+            '--detach',
```

Comment:
It would be good to leverage the same command line arguments shorthand as docker " -d, --detach"

```suggestion
            '-d',
            '--detach',
```